docs/tutorials/go-built-in.md

```diff
@@ -588,7 +588,7 @@ go 1.15
 
 require (
 	github.com/dgraph-io/badger v1.6.2
-	github.com/tendermint/tendermint v0.34.0-rc4
+	github.com/tendermint/tendermint v0.34.0
```

Comment:
This should point to master. It wont work with 0.34 because SetOption is gone. Was trying to fix this but now stuck in go mod `go get: inconsistent versions:` land